unfetter-discover/unfetter-store
# Issue 661

Partially addresses unfetter-discover/unfetter@issue-661

Added in getById and delete for capabilities, and added tests for get, getById, add, update, and delete of capabilities.  Add and update APIs are coming in a subsequent PR.

# To Test:

1. Pull this
2. Rebuild unfetter-store
3. Restart stack
4. Add this document to stix DB in Mongo:

{
    "_id" : "x-unfetter-capability--45256397-a396-40cd-b91a-95f39f99c24f",
    "stix" : {
        "type" : "x-unfetter-capability",
        "id" : "x-unfetter-capability--45256397-a396-40cd-b91a-95f39f99c24f",
        "created_by_ref" : "identity--4ac44385-691d-411a-bda8-027c61d68e99",
        "created" : "2016-09-14T17:56:37.030Z",
        "modified" : "2016-09-14T17:56:37.030Z",
        "name" : "Symantec Antivirus Corporate Edition",
        "description" : "Anti virus software",
        "version" : 10.2,
        "category" : "Antivirus",
        "external_references" : []
    }
}

5. Go to API Explorer
6. See STIX-Unfetter Capability section at the top
5. Try out get, getById("x-unfetter-capability--45256397-a396-40cd-b91a-95f39f99c24f"), and delete("x-unfetter-capability--45256397-a396-40cd-b91a-95f39f99c24f")
